### PR TITLE
New version: Microsoft.OneDrive.Internal.Slow version 22.225.1026.0001

### PR DESCRIPTION
**Pull request by vedantmgoyal9**
### Result: Installation Failed
|             | Manifest                 | Add/Remove Programs                   |
| ----------- | ------------------------ | ------------------------------------- |
| Name        | Microsoft OneDrive Internal Slow |     |
| Version     | 22.225.1026.0001     |  |
| Publisher   | Microsoft Corporation   |       |
| ProductCode |           |     |
###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/86638)

### CODE REVIEW

**Review by ItzLevvie — APPROVED** [review]

File: manifests/m/Microsoft/OneDrive/Internal/Slow/22.225.1026.0001/Microsoft.OneDrive.Internal.Slow.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  SilentWithProgress: /silent
+Installers:
+- Architecture: x64
+  InstallerUrl: https://oneclient.sfx.ms/Win/MsitSlow/22.225.1026.0001/arm64/OneDriveSetup.exe
+  InstallerSha256: E35C583CA23AF164FF91D36CD299C9FE006FDF7CABEC547ED5CBA5BE262A441C
+- Architecture: x86
+  InstallerUrl: https://oneclient.sfx.ms/Win/MsitSlow/22.225.1026.0001/amd64/OneDriveSetup.exe
+  InstallerSha256: 6513912FB6DAA99ABD1062055F6D9E0FD626A599336BF45FEFEB2631EEB1F929
+- Architecture: arm64
+  InstallerUrl: https://oneclient.sfx.ms/Win/MsitSlow/22.225.1026.0001/OneDriveSetup.exe
```
Comment (stephengillie) [inline manifests/m/Microsoft/OneDrive/Internal/Slow/22.225.1026.0001/Microsoft.OneDrive.Internal.Slow.installer.yaml:18]: InstallerUrl: https://oneclient.sfx.ms/Win/MsitSlow/22.225.1026.0001/arm64/OneDriveSetup.exe

File: manifests/m/Microsoft/OneDrive/Internal/Slow/22.225.1026.0001/Microsoft.OneDrive.Internal.Slow.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+InstallerType: exe
+InstallerSwitches:
+  Silent: /silent
+  SilentWithProgress: /silent
+Installers:
+- Architecture: x64
+  InstallerUrl: https://oneclient.sfx.ms/Win/MsitSlow/22.225.1026.0001/arm64/OneDriveSetup.exe
+  InstallerSha256: E35C583CA23AF164FF91D36CD299C9FE006FDF7CABEC547ED5CBA5BE262A441C
+- Architecture: x86
+  InstallerUrl: https://oneclient.sfx.ms/Win/MsitSlow/22.225.1026.0001/amd64/OneDriveSetup.exe
```
Comment (stephengillie) [inline manifests/m/Microsoft/OneDrive/Internal/Slow/22.225.1026.0001/Microsoft.OneDrive.Internal.Slow.installer.yaml:15]: InstallerUrl: https://oneclient.sfx.ms/Win/MsitSlow/22.225.1026.0001/OneDriveSetup.exe

File: manifests/m/Microsoft/OneDrive/Internal/Slow/22.225.1026.0001/Microsoft.OneDrive.Internal.Slow.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+PackageIdentifier: Microsoft.OneDrive.Internal.Slow
+PackageVersion: 22.225.1026.0001
+InstallerType: exe
+InstallerSwitches:
+  Silent: /silent
+  SilentWithProgress: /silent
+Installers:
+- Architecture: x64
+  InstallerUrl: https://oneclient.sfx.ms/Win/MsitSlow/22.225.1026.0001/arm64/OneDriveSetup.exe
```
Comment (stephengillie) [inline manifests/m/Microsoft/OneDrive/Internal/Slow/22.225.1026.0001/Microsoft.OneDrive.Internal.Slow.installer.yaml:12]: https://oneclient.sfx.ms/Win/MsitSlow/22.225.1026.0001/amd64/OneDriveSetup.exe

**Pull request merged**